src/steamship/agents/llms/openai.py

```diff
@@ -38,3 +38,52 @@ def complete(self, prompt: str, stop: Optional[str] = None) -> List[Block]:
         action_task = self.generator.generate(text=prompt, options=options)
         action_task.wait()
         return action_task.output.blocks
+
+
+# TODO(dougreid): should this be a an extension of a BaseOpenAI or similar? Or completely separate?
+class ChatOpenAI(ConversationalLLM):
+    """LLM that uses Steamship's OpenAI plugin to generate completions.
+
+    NOTE: By default, this LLM uses the `gpt-3.5-turbo` model. Valid model
+    choices are `gpt-3.5-turbo` and `gpt-4`.
+    """
+
+    generator: PluginInstance
+    client: Steamship
+
+    def __init__(
+        self, client, model_name: str = "gpt-4-0613", temperature: float = 0.4, *args, **kwargs
```

Comment:
```suggestion
        self, client, model_name: str = "gpt-4", temperature: float = 0.4, *args, **kwargs
```

`gpt-4-0613` is now the default.

Reply:
I think version-pegging it anyway is a good idea, since it makes it more clear to users what they're getting.

Reply:
Also looks like we'll need to update the comment on line 61